build_debian.sh

```diff
@@ -674,8 +674,12 @@ if [[ $SECURE_UPGRADE_MODE == 'dev' || $SECURE_UPGRADE_MODE == "prod" && $SONIC_
             exit 1
         fi
 
-        sudo $SECURE_UPGRADE_PROD_SIGNING_TOOL $CONFIGURED_ARCH $FILESYSTEM_ROOT $LINUX_KERNEL_VERSION $OUTPUT_SEC_BOOT_DIR
-        
+        sudo $sonic_su_prod_signing_tool -a $CONFIGURED_ARCH \
```

Comment:
>-a  [](http://example.com/codeflow?start=41&length=3)

will these new options like `-a` working for each vendor?

Reply:
It aligns the prod secure script with the way the dev secure scripts works.
If the vendors will take as an example the dev signature script their script will work.